simulation_ws/src/rl-agent/markov/environments/mars_env.py

```diff
@@ -355,27 +366,27 @@ def reward_function(self):
         '''
         
         # Corner boundaries of the world (in Meters)
-        STAGE_X_MIN = -44.0
-        STAGE_Y_MIN = -25.0
-        STAGE_X_MAX = 15.0
+        STAGE_X_MIN = -46
+        STAGE_Y_MIN = -22.0
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

For mathematical calculations, we recommend using the `decimal` module instead of the `float` datatype where any of the following is required - 
 1. exact decimal representation
 2. control over precision
 3. control over rounding to meet legal or regulatory requirements
 4. tracking of significant decimal places
 5. applications where the user expects the results to match calculations done by hand

[Learn more](https://docs.python.org/3/tutorial/stdlib2.html#decimal-floating-point-arithmetic)

Similar issue at line numbers 431, 436, 474, 476, 494, and 496.